src/meta/src/barrier/command.rs

```diff
@@ -175,6 +175,8 @@ pub enum Command {
     RescheduleFragment {
         reschedules: HashMap<FragmentId, Reschedule>,
         table_parallelism: HashMap<TableId, TableParallelism>,
+        // should contain the actor ids in upstream and downstream fragment of `reschedules`
+        fragment_actors: HashMap<FragmentId, HashSet<ActorId>>,
```

Comment:
Minor comment, I suppose `RescheduleFragment` is actually `RescheduleFragments`? Hence we have a hashmap here, rather than just the actor ids in upstream and downstream fragment of `reschedules`.

Reply:
Also why we refactor  and add the field `fragment_actors` here?

Reply:
I guess it is to make [to_mutation](https://github.com/risingwavelabs/risingwave/pull/14887/files#diff-30b031080778ca36f6e37c4769dd8941f9555643e0bab4f357c417a5093efc19R356) non-async

Reply:
Yes. It's for making it non-async.